### PR TITLE
Add new flash message type `info`

### PR DESCRIPTION
**Pull request by fstoerkle**
As discussed in #1220, we want to add a new `flashMessage` type `info` in order to show messages that are neither an error nor a success message.

### CODE REVIEW

File: src/core/index.js [hunk 1]
```diff
@@ -563,8 +575,9 @@ export function overrideOptions(m, opts) {
   }
 
   if (opts.flashMessage) {
-    const key = 'success' === opts.flashMessage.type ? 'globalSuccess' : 'globalError';
-    m = tset(m, key, opts.flashMessage.text);
+    const { type } = opts.flashMessage;
```
Comment (luisrudge) [inline src/core/index.js]: this will throw if flashMessage.type is undefined. use `error` as a default (+ a test)

**Comment by luisrudge** [comment]
Thanks @fstoerkle. Just left a single comment on something I found. 🎉

File: src/core/index.js (hunk 1 above)
Comment (fstoerkle) [reply src/core/index.js]: Makes sense.

However, I just had a look at the tests and there is one test case explicitly asserting that Lock should fail if there is no `type` set on a `flashMessage`:
https://github.com/auth0/lock/blob/master/test/show_with_flash_message.test.js#L19
This kind of contradicts your suggestion to make the `type` optional and falling back to `error` as default ` type`.

I'm fine with both options, what do you prefer?

Comment (luisrudge) [reply src/core/index.js]: You're right. I didn't remember that 😝 
Let's leave it like this. I'll test the PR today and merge it. Thanks! 🎉

Comment (fstoerkle) [reply src/core/index.js]: @luisrudge Hi Luis, did you manage to take a look at this again in the meanwhile?

**Comment by luisrudge** [comment]
Ops, sorry! Looking at it now.

**Comment by luisrudge** [comment]
Thanks for the PR 🎉 
It will be in the next release

**Comment by fstoerkle** [comment]
🚀  Sweet, thanks for taking care of this!

**Pull request merged**